app/experimenter/nimbus-ui/schema.graphql

```diff
@@ -278,7 +278,7 @@ type ProjectType {
 }
 
 type Query {
-  experiments(offset: Int, limit: Int, status: NimbusExperimentOptionalStatus): [NimbusExperimentType]
+  experiments(status: NimbusExperimentOptionalStatus): [NimbusExperimentType]
```

Comment:
Possibly related to previous comment above, but not sure that we're using the `status` attribute anymore.